libraries-5/src/test/java/com/baeldung/sootup/AnalyzeUnitTest.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.sootup;
+
+import org.junit.jupiter.api.Test;
+import sootup.core.inputlocation.AnalysisInputLocation;
+import sootup.java.bytecode.inputlocation.JavaClassPathAnalysisInputLocation;
+import sootup.java.bytecode.inputlocation.JrtFileSystemAnalysisInputLocation;
+import sootup.java.bytecode.inputlocation.OTFCompileAnalysisInputLocation;
+import sootup.java.core.views.JavaView;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class AnalyzeUnitTest {
+    @Test
+    void whenAnalyzingTheJVM_thenWeCanListClasses() {
```

Comment:
We follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case
```suggestion
    void whenAnalyzingTheJvm_thenWeCanListClasses() {
```